src/Mollie/WC/Payment/Order.php

```diff
@@ -193,14 +193,17 @@ public function getPaymentRequestData( $order, $customer_id ) {
             $paymentRequestData['payment']['cardToken'] = $cardToken;
         }
 
-        $applePayToken = $_POST['token'];
-        $applePayToken = filter_var($applePayToken, FILTER_SANITIZE_STRING);
-        if($applePayToken && isset($paymentRequestData['payment'])){
-            $encodedApplePayToken = json_encode($applePayToken);
-            $paymentRequestData['payment']['applePayPaymentToken'] = $encodedApplePayToken;
+        if($_POST['token']){
```

Comment:
Better sanitize it first, I suggest to use `filter_input` like so:
`$applePayToken = filter_input(INPUT_POST, 'token', FILTER_SANITIZE_STRING);`